dotcom-rendering/src/components/ImageComponent.tsx

```diff
@@ -386,6 +414,9 @@ export const ImageComponent = ({
 					/>
 				)}
 
+				{isTimeline && isMainMedia && role === 'showcase' && (
+					<span css={timelineBulletStyles} />
```

Comment:
Would `div` or `aside` be slightly more semantic? Are we making use of the fact that this is an inline element?

Reply:
Sure I'll change it to a div. I'll add an `aria-hidden="true"` as well